README.md

```diff
@@ -25,6 +25,7 @@ Optuna can dynamically construct the search spaces for the hyperparameters.
 
 ## :loudspeaker: News
 
+* **Aug 28, 2024**: We posted [an article about `INGO`](https://medium.com/optuna/a-natural-gradient-based-optimization-algorithm-registered-on-optunahub-0dbe17cb0f7d), a sampler newly supported in [OptunaHub](https://hub.optuna.org/samplers/implicit_natural_gradient/).
 * **Aug 20, 2024**: We posted [an article](https://medium.com/optuna/file-management-during-llm-large-language-model-trainings-by-optuna-v4-0-0-artifact-store-5bdd5112f3c7) about [`ArtifactStore`](https://optuna.readthedocs.io/en/stable/reference/artifacts.html) officially supported from Optuna 4.0.
 * **Aug 15, 2024**: We posted [an article about `CatCMA`](https://medium.com/optuna/introduction-to-catcma-in-optunahub-ffa316309cb8), a sampler newly supported in [OptunaHub](https://hub.optuna.org/samplers/catcma/).
```

Comment:
@eukaryo Could you change the URL linking in the same way as the suggestion above?

Reply:
Done!